### PR TITLE
Tutorial/layout components

### LINKED ISSUE #529: docs(tutorial): Section about Layout components
### Subject

Tutorial

### Description

Add a new section to the tutorial about the Layout Components of Chakra UI.

### PR DESCRIPTION
**Pull request by estheragbaje**
Closes #529 

## 📝 Description

Created tutorial section for layout components

## ⛳️ Current behavior (updates)

We plan to populate the tutorial section.

## 🚀 New behavior

N/A

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: configs/sandpack-contents/tutorial/package.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const packageJson = `
   "main": "src/index.js",
   "dependencies": {
     "@chakra-ui/react": "1.8.6",
+    "@chakra-ui/icons": "1.1.7",
```
Comment (noobinthisgame) [inline configs/sandpack-contents/tutorial/package.ts:13]: I assume this dependency is one you want to use in your tutorial? You need to add it here for it to work properly in the sandbox: https://github.com/chakra-ui/chakra-ui-docs/blob/main/src/components/tutorial/tutorial-container.tsx#L85-L91

Comment (noobinthisgame) [reply configs/sandpack-contents/tutorial/package.ts:13]: This package.json is just an example file and has no real impact on the sandbox.

File: pages/tutorial/layout-components.mdx [hunk 2]
```diff
@@ -3,4 +3,367 @@ title: 'Layout Components'
 description: 'TBD'
```
Comment (noobinthisgame) [inline pages/tutorial/layout-components.mdx:3]: Kindly add a small description here :)

**Review by noobinthisgame — CHANGES_REQUESTED** [review]
Thanks @estheragbaje for your work on this ! I think it looks really good already ! 

What I would love to see is a little more usage of the codesandbox on the right side. Maybe you could put a more advanced example of layout components usage as initial content for the users to have something to experiment with. Maybe something where you use multiple components. You can expand this https://github.com/chakra-ui/chakra-ui-docs/blob/main/layouts/tutorial.tsx#L85-L98 with a new `App.tsx`,  no need for the `package.json` :)

**Comment by segunadebayo** [comment]
Closing this since we're no longer shipping Tutorials for now.

**Pull request closed without merging**